app/models/step_by_step_page.rb

```diff
@@ -1,5 +1,6 @@
 class StepByStepPage < ApplicationRecord
   STATUSES = %w(
+    approved
```

Comment:
I agree, however I kept this consistent with `in_review` which doesn't mention 2i.

I think the statuses should be stored in the database as integers anyway and use Rails enum feature to refer to these as strings in the code.  That way, it would be easy to change the status text (e.g. when we implement factcheck) without having to do a database migration.  I'll see how much work would be involved in changing this value.